Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM python:3.7.3-slim
+
+ENV PYTHONUNBUFFERED=1
+ENV PYTHONDONTWRITEBYTECODE=1
+ENV AIRFLOW_HOME=/src/airflow
```

Comment:
`src` indicates source files in Unix speak, but there are installed libraries, logs, etc. in here; go for just `/airflow` like other applications (or [`/app`](https://github.com/libero/browser/blob/master/Dockerfile#L31))

Reply:
updated to /airflow.